deprecation-policy.md

```diff
@@ -17,11 +17,14 @@ A breaking change is when the primary functionality of a feature changes in a wa
 
 A "deprecation event" would coincide with a release. 
 
-1. we document the deprecation in 
+1. we document the deprecation in the following places if applicable 
 a.) docs
 b.) release notes 
-c.) command help (if applicable)
-d.) https://skaffold.dev/docs/references/yaml/ (if applicable)
+c.) command help 
+d.) logs
+e.) https://skaffold.dev/docs/references/yaml/ 
+f.) this doc
```

Comment:
name the doc instead of this doc in case this moves around.
f.) [Deprecation Policy Document](./deprecation-policy.md)